opstack/src/server/mod.rs

```diff
@@ -1,8 +1,14 @@
 use std::{net::SocketAddr, sync::Arc, time::Duration};
 
-use alloy::primitives::Address;
-use axum::{extract::State, routing::get, Json, Router};
+use alloy::{primitives::Address, rpc::types::EIP1186AccountProofResponse};
+use axum::{
+    extract::{Path, State},
+    http::StatusCode,
+    routing::get,
+    Json, Router,
+};
 use eyre::Result;
+use revm::primitives::B256;
```

Comment:
nit: lets use the alloy variant of this for clarity (should be same as it is just a reexport)